packages/terra-consumer-nav/src/Nav.jsx

```diff
@@ -3,23 +3,81 @@ import PropTypes from 'prop-types';
 import classNames from 'classnames/bind';
 import QuickLinks from './components/quick-links/QuickLinks';
 import NavItems from './components/nav-items/NavItems';
+import NavHelp from './components/nav-help/NavHelp';
+import UserProfile from './components/user-profile/UserProfile';
+import OutlineInfo from './icons/OutlineInfo';
+import UserIcon from './icons/UserIcon';
+import OutlineEllipses from './icons/OutlineEllipses';
 import styles from './Nav.scss';
 
 const cx = classNames.bind(styles);
 
 const propTypes = {
```

Comment:
This is a container component that contains the navigation that includes profile, quick links & so on and the container component that contains the page content. Agree, Layout would be a more relevant name for this component.